docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,80 @@
+* Creates the response console, an interface that enables you to take actions on specific hosts ({pull}135360[#135360], {pull}134520[#134520]).
+* Enables a new method for the Task Manager API: `bulkUpdateSchedules`, which enables you to update the execution timing of `idle` tasks ({pull}132637[#132637]).
+* Includes integration policy errors and statuses in both {fleet} and {elastic-security} to help troubleshoot when an {agent} has an `Unhealthy` status.
+* Adds Attack surface reduction protections feature to reduce vulnerabilities on Windows endpoints. Credential hardening prevents attackers from stealing credentials stored in Windows system process memory.
+* Adds an endpoint self-healing feature to roll back file changes and processes on Windows endpoints when a prevention alert is generated by enabled protection features.
+* Adds the ability to run query packs as live queries ({pull}132198[#132198]).
+* Provides support for process, file, and network events in Kubernetes. You must enable the session view data setting on your {endpoint-cloud-sec} integration policy to enrich these events with session data and Kubernetes metadata fields.
+
+[discrete]
+[[bug-fixes-8.4.0]]
+==== Bug fixes and enhancements
+* Updates the Network page's UI to match the Hosts and Users pages ({pull}137541[#137541], {pull}136913[#136913]).
+* Fixes an error that could occur when you tried to apply an index pattern to rules using the bulk action option ({pull}134664[#134664]).
+* Enhances rule previews with configurable rule intervals and look-back times ({pull}137102[#137102]).
+* Enhances the `status pending` badge for endpoint actions with a detailed status when you hover on it ({pull}136966[#136966]).
+* Turns grouped navigation on by default ({pull}136819[#136819]).
+* Adds a confirmation dialog to bulk rule export ({pull}136418[#136418]).
+* Adds index pattern information to the Inspect panel ({pull}136407[#136407]).
+* Adds a custom dashboards table to the Dashboards page ({pull}136221[#136221], {pull}136671[#136671]).
+* Fixes a performance issue with alerts that have large fields ({pull}135956[#135956]).
+* Updates the rule exceptions UI ({pull}135255[#135255]).
+* Fixes performance issues with rules management ({pull}135311[#135311]).
+* Allows you to define a fallback `@timestamp` when you've defined a timestamp override ({pull}135116[#135116]).
+* Enhances the host risk score modal UI ({pull}133708[#133708]).
+* Updates the lists index template to use new logic ({pull}133067[#133067]).
+* Adds event filters to event correlation rules ({pull}132507[#132507]).
+* Allows you to define a data view or index pattern for rules ({pull}130929[#130929]).
+* Creates a single visualization pane on the Alerts page, and adds a treemap view that shows the distribution of alerts as nested, proportionally-sized tiles. ({pull}126896[#126896]).
```

Comment:
nit: I would change view to visualization

Reply:
```suggestion
* Creates a single visualization pane on the Alerts page, and adds a treemap visualization that shows the distribution of alerts as nested, proportionally-sized tiles ({pull}126896[#126896]).
```